go/Dockerfile

```diff
@@ -1,18 +1,19 @@
 FROM golang:1.14.1 as delve
 
 RUN curl --location --output delve-1.4.0.tar.gz https://github.com/go-delve/delve/archive/v1.4.0.tar.gz \
-  && tar xzf delve-1.4.0.tar.gz
+  && tar xzf delve-1.4.0.tar.gz \
+  && mv delve-1.4.0 delve-source
```

Comment:
does delve have enough other version info to debug this?
should we write a version file down next to it <-- or is this of no value?

Reply:
I'm not sure what you mean by "_debug this_"?  I don't think it's of value — we know from the curl what version is being downloaded.

If the patch fails to apply, it exits and the image doesn't build as can be seen in the [error log](https://travis-ci.org/github/GoogleContainerTools/container-debug-support/builds/693328155#L266) from #39.

Reply:
Yeah it's not really a problem, I think I was overthinking it and confused myself. I was wondering how we'll know which delve was being used when a skaffold debug of go app goes wrong. But I think there's all kinds of other ways to get that information.

Reply:
We can just ask delve:

```console
$ docker run gcr.io/gcp-dev-tools/duct-tape/go /duct-tape/go/bin/dlv version
Delve Debugger
Version: 1.4.1
Build: $Id: bda606147ff48b58bde39e20b9e11378eaa4db46 $
```